### PR TITLE
update  Windows Analytics in the Azure Portal

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #3367
changed 6 screenshots

### CODE REVIEW

**Comment by mypil** [comment]
@MaratMussabekov - Please make sure that the screenshots/graphics in this PR aligns to the guidelines found in the [Office Content Contributor Toolkit](https://support.office.com/en-us/article/create-graphics-for-office-e250da33-5edd-46ff-81a7-8913adac36fb?ui=en-US&rs=en-US&ad=US). Otherwise, you may have to close this and create a new PR. Thank you.

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/windows-analytics-azure-portal.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ ms.topic: article
 
 # Windows Analytics in the Azure Portal
 
-Windows Analytics uses Azure Log Analytics (formerly known as Operations Management Suite or OMS), a collection of cloud-based servicing for monitoring and automating your on-premises and cloud environments.
+Windows Analytics uses Azure Log Analytics workspaces (formerly known as Operations Management Suite or OMS), a collection of cloud-based servicing for monitoring and automating your on-premises and cloud environments.
```
Comment (nenonix) [inline windows/deployment/update/windows-analytics-azure-portal.md]:
```suggestion
Windows Analytics uses Azure Log Analytics workspaces (formerly known as Operations Management Suite or OMS), a collection of cloud-based services for monitoring and automating your on-premises and cloud environments.
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix , you recommendation was applied, thank you!

**Comment by mypil** [comment]
@lomayor @jaimeo - Can you please assist to check, approve and merge the content of this PR based on issue #3367?

Thank you.

**Comment by jaimeo** [comment]
The PM for this specifically said this change was unnecessary, but thanks anyway.

**Pull request merged**